models/iaf_psc_alpha_multisynapse.cpp

```diff
@@ -44,20 +44,50 @@
  * Recordables map
  * ---------------------------------------------------------------- */
 
-nest::RecordablesMap< nest::iaf_psc_alpha_multisynapse >
-  nest::iaf_psc_alpha_multisynapse::recordablesMap_;
-
 namespace nest
 {
 // Override the create() method with one call to RecordablesMap::insert_()
 // for each quantity to be recorded.
 template <>
 void
-RecordablesMap< iaf_psc_alpha_multisynapse >::create()
+DynamicRecordablesMap< iaf_psc_alpha_multisynapse >::create(
+  iaf_psc_alpha_multisynapse& host )
+{
+  // use standard names wherever you can for consistency!
+  insert( names::V_m,
+    host.get_data_access_functor( iaf_psc_alpha_multisynapse::State_::V_M ) );
+
+  insert( names::currents,
+    host.get_data_access_functor( iaf_psc_alpha_multisynapse::State_::I ) );
```

Comment:
I think `currents` was never a good name and it is only used in this one model. I suggest to replace it with `I_syn`.